database/database.go

```diff
@@ -42,9 +38,9 @@ var (
 
 	// ErrUserNotFound is returned when we can't find the user in question.
 	ErrUserNotFound = errors.New("user not found")
-	// ErrEmailAlreadyUsed is returned when we try to use an email to either
-	// create or update a user and another user already uses this email.
-	ErrEmailAlreadyUsed = errors.New("email already in use by another user")
+	// ErrUserAlreadyExists is returned when we try to use a sub to create a
+	// user and a user already exists with this sub. This should never happen.
+	ErrUserAlreadyExists = errors.New("sub already belongs to an existing user")
```

Comment:
Will readers of this error know what "sub" means?

Reply:
+1

Reply:
I thought it was a well-known reference to identity string in JWTs. But I can change it to something clearer. The issue is that I can't say `id` here because that would imply the user id in Mongo, not the external user id we get from ORY.

Any suggestions?

Reply:
it might be well known to people who know jwt well :)

Reply:
I never heard about it, and I knew what claims were, never encountered it before 🤷‍♂️

Reply:
Changed.